src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -485,13 +485,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			fluid.generateCondition = emptyLists ? null : new Procedure("condition1");
 			fluid.getModElement().putMetadata("gb", fluid.generateBucket);
 			return fluid;
-		} else if (ModElementType.COMMAND.equals(modElement.getType())) {
-			Command command = new Command(modElement);
-			command.commandName = modElement.getName();
-			command.permissionLevel = getRandomItem(random, new String[] { "No requirement", "1", "2", "3", "4" });
-			command.onCommandExecuted = new Procedure("procedure2");
-			return command;
-		} else if (ModElementType.KEYBIND.equals(modElement.getType())) {
+		}else if (ModElementType.KEYBIND.equals(modElement.getType())) {

```

Comment:
The change there is redundant.
```suggestion
		} else if (ModElementType.KEYBIND.equals(modElement.getType())) {
```